packages/strapi-admin/ee/admin/containers/Roles/CreatePage/Permissions/utils/tests/createDefaultPluginsFormFromLayout.test.js

```diff
@@ -0,0 +1,227 @@
+import createDefaultPluginsFormFromLayout, {
+  createSubCategoryForm,
+  createChildrenDefaultForm,
+} from '../createDefaultPluginsFormFromLayout';
+
+const conditions = [
+  {
+    id: 'admin::is-creator',
+    displayName: 'Is creator',
+    category: 'default',
+  },
+  {
+    id: 'admin::has-same-role-as-creator',
+    displayName: 'Has same role as creator',
+    category: 'default',
+  },
+];
+
+describe('ADMIN | COMPONENTS | Permissions | utils', () => {
+  describe('createSubCategoryForm', () => {
```

Comment:
Can you write a test with at least one leaf true?

Reply:
Of course, I'll update the tests in #9420